I/iso_codes/build_tarballs.jl

```diff
@@ -3,7 +3,7 @@
 using BinaryBuilder
 
 name = "iso_codes"
-version = v"4.15.0"
+version = v"4.15.1"
```

Comment:
@giordano do we still need to do this version bump or will the build number take care of the issue?

Reply:
If we change the julia compat below we must have a different version number